zerver/openapi/zulip.yaml

```diff
@@ -5259,27 +5259,30 @@ paths:
         - name: narrow
           in: query
           description: |
-            The narrow where you want to fetch the messages from. See how to
+            The narrow filter where you want to fetch the messages from. See how to
             [construct a narrow](/api/construct-narrow).
 
-            Note that many narrows, including all that lack a `stream`
-            or `streams` operator, search the user's personal message
-            history. See [searching shared
-            history](/help/search-for-messages#searching-shared-history)
+            Note that many narrows, including all that lack a `stream` or `streams`
+            operator, search the user's personal message history. See [searching
+            shared history](/help/search-for-messages#searching-shared-history)
             for details.
 
-            For example, if you would like to fetch messages from all public streams instead
-            of only the user's message history, then a specific narrow for
-            messages sent to all public streams can be used:
+            For example, if you would like to fetch messages from all public
+            streams instead of only the user's message history, then a specific
+            narrow for messages sent to all public streams can be used:
             `{"operator": "streams", "operand": "public"}`.
 
-            Newly created bot users are not usually subscribed to any
-            streams, so bots using this API should either be
-            subscribed to appropriate streams or use a shared history
-            search narrow with this endpoint.
+            Newly created bot users are not usually subscribed to any streams,so
```

Comment:
nit:
```suggestion
            Newly created bot users are not usually subscribed to any streams, so
```

Reply:
But actually better: this paragraph isn't changing at all except for rewrapping, right? Probably better to just leave it untouched, then. Same for the two paragraphs before it.

Reply:
Yeah, instead I'm making a note to come back and clean up some of the line wrapping later.